apps/sap-commerce-cloud/src/index.tsx

```diff
@@ -41,8 +41,8 @@ const getApplicationInterfaceKey = async (): Promise<boolean | string> => {
 
 init(async (sdk) => {
   const root = document.getElementById('root');
-  const isTestEnv = config.isTestEnv;
-  const sapApplicationInterfaceKey = !isTestEnv ? await getApplicationInterfaceKey() : '';
+  //const isTestEnv = config.isTestEnv;
```

Comment:
Can it be removed, or is this coming back into action at a later point? LGTM otherwise nothing blocking

Reply:
Coming back into action at some point, we've reached out to SAP to help diagnose as it doesn't look to be an issue on our side.